README.md

```diff
@@ -32,7 +32,7 @@ The [CHARTER.md](CHARTER.md) outlines the scope and governance of our group acti
 
 ## Meeting times
 
-[Google Meet](https://meet.google.com/nid-mxeg-xwt) every other Tuesday at 16:00 GMT from Aug 11. 
+[Zoom](https://zoom.us/j/93752451148?pwd=WW5YaHJuMW1MUFNnazV4WVhTRDlqZz09) every other Tuesday at 16:00 GMT from Aug 11.
```

Comment:
This is different from the one in the current meeting invite: https://zoom.us/j/96055397788?pwd=bzFxYjNrUTZvNDFLdjVCZHQyWDhNdz09
Was that on purpose? I'm pretty sure I used this one yesterday...

Reply:
> This is different from the one in the current meeting invite: https://zoom.us/j/96055397788?pwd=bzFxYjNrUTZvNDFLdjVCZHQyWDhNdz09 Was that on purpose? I'm pretty sure I used this one yesterday...

Ah, sorry, not on purpose. This is actually the link to the (old) Kubernetes Multitenancy WG meeting which happens to be next to the OpenSSF security tooling meeting in my calendar and I copied the link from the wrong meeting... (see https://docs.google.com/document/d/1fj3yzmeU2eU8ZNBCUJG97dk_wC7228-e_MmdcmTNrZY/edit#heading=h.7nir5parfd2n , old Zoom link).

Good catch and sorry for the confusion. Not a good way of introducing oneself.